### ISSUE TITLE
The Two Project Examples I have tried are not working properly

### ISSUE THREAD

**Issue by sharon13m** [Mobsya/thymio-scratchx#10]
The program plays, 20 points on the score board and nothing else.
http://scratchx.org/?url=https://aseba-community.github.io/thymio-scratchx/activities/thymio-space_en.sbx&lang=en#scratch

Piano makes a sound upon pressing the flag, but nothing more
http://scratchx.org/?url=https://aseba-community.github.io/thymio-scratchx/activities/piano.sbx&lang=en#scratch

**Comment by mbonani** [comment]
First program it is like that, there is no end until know

Second works fine for me. What show the value of the sensors on Thymio at right, you have really to put your finger on the sensor to play sounds. Also is your Thymio playing sound normally?

**Comment by sharon13m** [comment]
Hi Michael,
I figured out why the Piano isn't working for me, my Thymio doesn't ever get over 4300 with its sensor readings.  I changed it to 4000 and it worked.  I recommend you change the program to 3900 to account for differently calibrated Thymios.

For the First program, you have that program linked to the English page.. but it is in French.  And frankly.. I don't know French and have no idea what is going on.  So I'd recommend either having an English version as well, or removing that program from the english page altogether.
